src/plugins/kibana_react/public/field_button/field_button.scss

```diff
@@ -38,6 +38,7 @@
   padding: $euiSizeS;
   display: flex;
   align-items: flex-start;
+  line-height: normal;
```

Comment:
:information_source: This is needed, due to the removed empty callback above, to make the now not `button` element look the same as the `button` it was beforehand.